src/api/goods.ts

```diff
@@ -3,17 +3,29 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export function getAll(): Promise<Good[] | null> {
   return fetch(API_URL)
     .then(response => response.json());
 }
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      if (!goods) {
```

Comment:
then if you handled Error check in `getAll()` method you can remove this null check